.goreleaser-pre.yml

```diff
@@ -86,3 +106,54 @@ brews:
 
     install: |
       bin.install "policy"
+
+dockers:
```

Comment:
Do we need dockers for the pre-release ?

Reply:
removed